### DISCUSSION TITLE
It is impossible to download an Onlyfans video

### DISCUSSION THREAD

**Discussion by dhimminni** [aclap-dev/video-downloadhelper#353] [Help and Q&A]
### Your question

I can't download any Onlyfans video, sometimes this message appears Native Port Disconnected and at other times I JUST DOWNLOAD THE AUDIO AND NOT THE VIDEO

### Addon & CoApp version and details

I can't download any Onlyfans video, sometimes this message appears Native Port Disconnected and at other times I JUST DOWNLOAD THE AUDIO AND NOT THE VIDEO

**Comment by jcfrog** [comment]
Hi,
Some reported onlyfans uses DRM, is it your case?

**Comment by dhimminni** [comment]
the DRM is released for viewing. The app only downloads the audio of
the video without the image or it gives a "Native Port Disconnected"
error. I haven't been able to download any videos so far.



Em seg., 1 de abr. de 2024 às 05:48, jcfrog ***@***.***>
escreveu:

**Comment by dhimminni** [comment]
The DRM is released for viewing. The app only downloads the audio of the video without the image or it gives a "Native Port Disconnected" error. I haven't been able to download any videos so far.

**Comment by jcfrog** [comment]
can you share your versions? https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by dhimminni** [comment]
version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: pt-BR
license: accepted
key (only 16 first characters): ea9323133db14dbf
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac13",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/isaiasmartins",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.1.1-tessus"
}

**Comment by Baloo9753** [comment]
Hello,
it seems the latest version and the beta can not load videos from payed onlyfans account anymore when DASH is the used file format ...
Using Win11pro latest release, Firefox latest release, licensed downloadhelper latest and Beta...
A file will be written in the nearly correct size, but it is not playable - also instead of saving the file in background, the standard player is opened without visible content ....

Any idea?
Thx Werner

**Comment by amko61** [comment]
I cant download DASH files too. I can convert them to mp4-files automatically, but that doesnt help :-( I'm happy for any suggestions!

**Comment by FDNY8231** [comment]
It's a paid program, but it WORKS ... CleverGet downloader is the name, and there IS A software registration fee to use it. It does however work with not only OnlyFans, but others as well like Fansly, Amazon, Hulu, Netflix .... There is a 3 VIDEO LIMIT on the free version. I downloaded three free ones and then did the registration. This is the ONLY piece of software I have found that works for OnlyFans.          https://www.cleverget.com/onlyfans-downloader

**Comment by mediadown** [comment]
You can try this specific tool for this website. It offers both batch and individual downloads, and there is a free trial available. https://github.com/mediadown/Onlyfans-Downloader

**Comment by DamonTsang986** [comment]
You can try some desktop onlyfans downloader tools such as this one: https://www.moyeasoft.com/downloader/onlyfans-downloader/. I have used it for months and it works all the way.
